megatron/model/rwkv/v6/rwkv.py

```diff
@@ -247,11 +247,11 @@ def __init__(self, neox_args, layer_number):
             self.time_maa_k = nn.Parameter(1.0 - torch.pow(ddd, ratio_1_to_almost0))
             self.time_maa_r = nn.Parameter(1.0 - torch.pow(ddd, ratio_1_to_almost0))
 
-        self.key = nn.Linear(neox_args.hidden_size, neox_args.dim_ffn, bias=False)
```

Comment:
I prefer `ffn` to `ff`. Also, why are we swapping the ordering of `dim` and `ffn` in the first place?